<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionArchivo_Nuevo"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionGuardar_Como"/>
   <addaction name="separator"/>
   <addaction name="actionEjecutar"/>
   <addaction name="actionEjecutar_Resaltado"/>
   <addaction name="separator"/>
   <addaction name="actionTransferir"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_5">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <action name="actionEjecutar_Resaltado">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/playsel.png</normaloff>:/playsel.png</iconset>
   </property>
   <property name="text">
    <string>Ejecutar Resaltado</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEjecutar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/play.png</normaloff>:/play.png</iconset>
   </property>
   <property name="text">
    <string>Ejecutar</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionTransferir">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/transfer2.png</normaloff>:/transfer2.png</iconset>
   </property>
   <property name="text">
    <string>Transferir</string>
   </property>
  </action>
  <action name="actionGuardar_Como">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/saveas.png</normaloff>:/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Guardar Como</string>
   </property>
  </action>
  <action name="actionArchivo_Nuevo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/newfile.png</normaloff>:/newfile.png</iconset>
   </property>
   <property name="text">
    <string>Archivo Nuevo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
